src/utils/pkg.ts

```diff
@@ -21,10 +21,10 @@ export function parse(input: string): PackageRequirement {
       const parts = v.split(".")
       const n = parts.length
       switch (n) {
-      case 0:
+      case 1:
```

Comment:
what's this all about then?

Reply:
split never returns 0 parts.  "6".split(".") returns ["6"]

Reply:
ofc. how did the tests pass?!?!?

Are we protected against `"".split('.')` that always catches me in js